lib/ansible/modules/cloud/vmware/vmware_cluster.py

```diff
@@ -34,139 +33,14 @@
     cluster_name:
       description:
       - The name of the cluster to be managed.
+      type: str
       required: yes
     datacenter:
       description:
       - The name of the datacenter.
+      type: str
       required: yes
       aliases: [ datacenter_name ]
-    enable_drs:
```

Comment:
I would say keep this code / functionalities as is and warn user when they try to use this. Providing correct module name for the given functionality.

This will not break existing playbooks and will allow users to migrate to new modules. Removing code abruptly will introduce chaos.

Reply:
OK, I understand what you mean. I've reverted the vmware_cluster module to the original version plus some deprecation warnings. But I don't like it, there are one or two things that somehow feel wrong. I do have an idea how to do this in a better way, but I'll have to think this out.

Reply:
I've added three new parameters: `ignore_drs`, `ignore_ha` and `ignore_vsan`. This way, people can get rid of the deprication warnings when using the new modules by making vmware_cluster ignore DRS, HA and VSAN configuration.

Since the default is `False`, these new parameters shouldn't break any existing playbooks.